### ISSUE TITLE
Can't get audio with more than 2 channel?

### ISSUE THREAD

**Issue by e33et** [flipswitchingmonkey/FlexASIO_GUI#26]
Hello flipswitchingmonkey,

Can't get audio to works with asio if I set more than 2 channel? and you say it shouldn't limit channel and my Sound Blaster X3 has 8 ch.

Otherwise it works as exptected!

Kind Regards
/e33et

**Comment by dechamps** [comment]
Hello,

Have you tried the instructions in [this FAQ](https://github.com/dechamps/FlexASIO/blob/master/FAQ.md#why-does-the-device-channel-countrouting-seem-wrong-with-some-backends)?

By the way this seems like the issue you're having is about FlexASIO, not FlexASIO GUI. If you can't get it to work, please [gather a log](https://github.com/dechamps/FlexASIO#logging) and attach it to a [new issue filed against the FlexASIO project](https://github.com/dechamps/FlexASIO/issues/new) to that I can take a look.

**Comment by e33et** [comment]
> Hello,
> 
> Have you tried the instructions in [this FAQ](https://github.com/dechamps/FlexASIO/blob/master/FAQ.md#why-does-the-device-channel-countrouting-seem-wrong-with-some-backends)?
> 
> By the way this seems like the issue you're having is about FlexASIO, not FlexASIO GUI. If you can't get it to work, please [gather a log](https://github.com/dechamps/FlexASIO#logging) and attach it to a [new issue filed against the FlexASIO project](https://github.com/dechamps/FlexASIO/issues/new) to that I can take a look.

Not really! But after tried 6 channels I've got 5.1 surround and can switch to Bluetooth and vice versa without being interrupted with device issues with Reason Studios and foobar2000 so I'm happy with your contribution!

Merry Xmas

Kind Regards
/e33ete
